python/pyiceberg/table/__init__.py

```diff
@@ -64,25 +67,326 @@
     import ray
     from duckdb import DuckDBPyConnection
 
+    from pyiceberg.catalog import Catalog
 
 ALWAYS_TRUE = AlwaysTrue()
 
 
+class Transaction:
```

Comment:
Do we plan to have another `MiltiTableTransaction` for transactions across tables?

Reply:
I'd rather expect a `List[Transaction]`